src/icons/address_book.rs

```diff
@@ -1,38 +1,39 @@
+use crate::IconWeight;
 /// GENERATED FILE
-
 use leptos::*;
-use crate::IconWeight;
 
 #[component]
 pub fn AddressBook(
-    #[prop(into, default = MaybeSignal::Static(IconWeight::Regular))] weight: MaybeSignal<IconWeight>,
-    #[prop(into, default = MaybeSignal::Static("1em".to_string()))] size: MaybeSignal<String>,
-    #[prop(into, default = MaybeSignal::Static("currentColor".to_string()))] color: MaybeSignal<String>,
-    #[prop(into, default = MaybeSignal::Static(false))] mirrored: MaybeSignal<bool>
+    #[prop(into, default = MaybeSignal::Static(IconWeight::Regular))] weight: MaybeSignal<
+        IconWeight,
+    >,
+    #[prop(into, default = TextProp::from("1em"))] size: TextProp,
```

Comment:
[`TextProp`](https://docs.rs/leptos/latest/leptos/struct.TextProp.html) is a bit more versatile than `MaybeSignal<String>` while also preserving performance.